test_scripts/API/CloseApplication/003_App_is_registered_with_lower_version_than_the_RPC_required.lua

```diff
@@ -0,0 +1,53 @@
+---------------------------------------------------------------------------------------------------
+--  Precondition: 
+--  1) Application with <appID> is registered on SDL with majorVersion = 5 and minorVersion = 0
+--  2) Specific permissions are assigned for <appID> with CloseApplication
+--
+--  Steps:
+--  1) Application sends a CloseApplication RPC request
+--
+--  Expected:
+--  1) SDL doesn't send CloseApplication RPC to the HMI and respond with INVALID_DATA to the Application
+---------------------------------------------------------------------------------------------------
+
```

Comment:
```suggestion
```

Reply:
@VjKlepikov Done in 66b5dec050ea98d28ec411f8f03e063830d079a4